Demos/Blazorise.Demo/Pages/Tests/VideoPage.razor

```diff
@@ -19,21 +19,32 @@
                 </Buttons>
                 <Button Color="Color.Primary" Clicked="@OnChangeVideoUrl">Change Video</Button>
                 <Button Color="Color.Primary" Clicked="@OnChangeCurrentTime">Go to 20 seconds</Button>
+                <Button Color="Color.Secondary" Clicked="@OnChangeQuality">Change Quality to 1080</Button>
             </CardBody>
             <CardBody>
-                <Video @ref="@videoRef" Source="@videoUrl" CurrentTime="@currentTime" />
+                <Video @ref="@videoRef" Source="@videoSource" CurrentTime="@currentTime" DefaultQuality="@quality" />
             </CardBody>
         </Card>
     </Column>
 </Row>
 @code {
     Video videoRef;
-    string videoUrl = "http://commondatastorage.googleapis.com/gtv-videos-bucket/sample/BigBuckBunny.mp4";
+    VideoSource videoSource = new VideoSource()
+    {
+        Medias = new ValueEqualityList<VideoMedia>
+        {
+            new VideoMedia("https://cdn.plyr.io/static/demo/View_From_A_Blue_Moon_Trailer-576p.mp4", "video/mp4", 576),
+            new VideoMedia("https://cdn.plyr.io/static/demo/View_From_A_Blue_Moon_Trailer-720p.mp4", "video/mp4", 720),
+            new VideoMedia("https://cdn.plyr.io/static/demo/View_From_A_Blue_Moon_Trailer-1080p.mp4", "video/mp4", 1080),
```

Comment:
Can we get some static helpers for the quality? Or we shouldn't bother? 720p ; 1080p ; etc...

Reply:
I'm not sure what that helper would do. Any idea?

Reply:
Just something like 
VideoPlayer.Quality.480p
VideoPlayer.Quality.720p
VideoPlayer.Quality.1080p
VideoPlayer.Quality.1440p
VideoPlayer.Quality.2160p
with some predefined common values.

Reply:
Video can have any size, so it is impossible to make it generic. Skipping this.

Reply:
It's an helper class with predefined int values. I'm not talking about making anything generic.